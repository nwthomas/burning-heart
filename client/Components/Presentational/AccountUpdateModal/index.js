export { default as AccountUpdateModal } from "./AccountUpdateModal";
