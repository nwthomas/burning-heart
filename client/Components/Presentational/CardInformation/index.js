export { default as CardInformation } from "./CardInformation";
