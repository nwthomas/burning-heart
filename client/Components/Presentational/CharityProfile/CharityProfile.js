import React, { Component } from "react";
import { View, Text } from "react-native";

const CharityProfile = ({ charity }) => {
  return (
    <View>
      <Text> textInComponent </Text>
    </View>
  );
};

export default CharityProfile;
