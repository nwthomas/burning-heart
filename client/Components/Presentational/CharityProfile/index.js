export { default as CharityProfile } from "./CharityProfile";
