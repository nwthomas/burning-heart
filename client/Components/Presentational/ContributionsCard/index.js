export { default as ContributionsCard } from "./ContributionsCard";
