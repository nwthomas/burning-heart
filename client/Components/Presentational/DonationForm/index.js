export { default as DonationForm } from "./DonationForm";
