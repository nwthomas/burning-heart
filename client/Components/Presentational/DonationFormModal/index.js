export { default as DonationFormModal } from "./DonationFormModal";
