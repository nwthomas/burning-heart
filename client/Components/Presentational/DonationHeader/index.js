export { default as DonationHeader } from "./DonationHeader";
