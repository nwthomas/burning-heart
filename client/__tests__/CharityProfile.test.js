import React from "react";
import "react-native";
import { CharityProfile } from "../Components/Presentational/CharityProfile";
import renderer from "react-test-renderer";

describe("<CharityProfile />", () => {
  it("should render the component correctly without crashing", () => {
    const helpers = renderer.create(<CharityProfile />);
  });

  it("should match the snapshot of the tree", () => {
    const tree = renderer.create(<CharityProfile />);
    expect(tree.toJSON()).toMatchSnapshot();
  });
});
