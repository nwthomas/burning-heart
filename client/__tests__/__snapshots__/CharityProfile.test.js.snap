// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CharityProfile /> should match the snapshot of the tree 1`] = `
<View>
  <Text>
     textInComponent 
  </Text>
</View>
`;
